pay-id-api-spec/pay-id.v1.yml

```diff
@@ -0,0 +1,333 @@
+          type: string
+        message:
+          $ref: '#/components/schemas/Invoice'
+        publicKeyType:
+          type: string
+        publicKeyData:
+          type: array
+          items:
+            type: string
+        publicKey:
+          type: string
+        signature:
+          type: string
+      required:
+        - messageType
+        - message
+        - publicKeyType
+        - publicKeyData
+        - publicKey
+        - signature
+    SignatureWrapper_Compliance:
+      title: SignatureWrapper Compliance Message
+      type: object
+      properties:
+        messageType:
+          type: string
+        message:
+          $ref: '#/components/schemas/Compliance'
+        publicKeyType:
+          type: string
+        publicKeyData:
+          type: array
+          items:
+            type: string
+        publicKey:
+          type: string
+        signature:
+          type: string
+      required:
+        - messageType
+        - message
+        - publicKeyType
+        - publicKeyData
+        - publicKey
+        - signature
+    Beneficiary:
+      title: Beneficiary
+      type: object
+      properties:
+        institutionName:
+          type: string
+        userLegalName:
+          type: string
+        userPhysicalAddress:
+          type: string
+        accountId:
+          type: string
+      required:
+        - institutionName
+    Value:
+      title: Value
+      type: object
+      properties:
+        amount:
+          type: string
+        scale:
+          type: string
+      required:
+        - amount
+        - scale
+    Originator:
+      title: Originator
+      type: object
+      properties:
+        userLegalName:
+          type: string
+        accountId:
+          type: string
+        userPhysicalAddress:
+          type: string
+        institutionName:
+          type: string
+        value:
+          $ref: '#/components/schemas/Value'
+        timestamp:
+          type: string
+      required:
+        - userLegalName
+        - accountId
+        - userPhysicalAddress
+        - institutionName
+        - value
+        - timestamp
+    TravelRule:
+      title: TravelRule
+      type: object
+      properties:
+        originator:
+          $ref: '#/components/schemas/Originator'
+        beneficiary:
+          $ref: '#/components/schemas/Beneficiary'
+      required:
+        - originator
+        - beneficiary
+    Compliance:
+      title: Compliance
+      type: object
+      properties:
+        type:
+          type: string
+        data:
+          $ref: '#/components/schemas/TravelRule'
+      required:
+        - type
+        - data
+    Receipt:
+      title: Receipt
+      type: object
+      properties:
+        invoiceHash:
+          type: string
+        transactionConfirmation:
+          type: string
+      required:
+        - invoiceHash
+        - transactionConfirmation
+openapi: '3.0.0'
+servers:
+  - url: 'https:/'
+    description: ''
```

Comment:
Does this mean that this `PayID` swaggger file is duplicated in `XpringKit`, `Xpring-JS`, and `Xpring4J`?

Reply:
It does for now. Eventually, I'd like this to live in a single place but I haven't gotten that chance yet.